python/pyarrow/feather.py

```diff
@@ -151,7 +151,18 @@ def write_feather(df, dest, compression=None, compression_level=None,
             df = df.to_dense()
 
     if _pandas_api.is_data_frame(df):
-        table = Table.from_pandas(df, preserve_index=False)
+        """
+        Feather v1 creates a new column in the resultant Table to
+        store index information if index type is not RangeIndex
+        """
```

Comment:
```suggestion
        # Feather v1 creates a new column in the resultant Table to
        # store index information if index type is not RangeIndex
```

Inside a function, we typically use `# ` for comment strings, and only use the triple quote for the function docstring (directly after the function `def` line)